src/cascade/executor/estimate_location.py

```diff
@@ -126,11 +130,11 @@ def retrieve_data(execution_context, local_settings, covariate_data_spec):
     data.study_id_to_name, data.country_id_to_name = find_covariate_names(
         execution_context, covariate_data_spec)
     # These are the draws as output of the parent location.
-    data.draws = None
+    data.draws = local_cache.get(f"fit-draws:{local_settings.grandparent_location_id}")
 
     # The parent can also supply integrands as a kind of prior.
     # These will be shaped like input measurement data.
-    data.integrands = None
+    data.integrands = local_cache.get(f"fit-integrands:{local_settings.grandparent_location_id}")
```

Comment:
No matching set?

Reply:
We aren't generating them yet, but that's a way to get them.